src/python/dxpy/sugar/__init__.py

```diff
@@ -15,6 +15,15 @@
 #   under the License.
 import dxpy
 from functools import wraps
+import re
+
+
+MEM_RE = re.compile(r"^MemAvailable:[\s]*([0-9]*) kB")
+MEM_CONVERSIONS = {
```

Comment:
Wondering if we should use a more descriptive name so people know it's assumed KiB. Maybe `MEM_KiB_CONVERSIONS`.

Up to you here.